### ISSUE TITLE
Actual food/hr ignores Gaia's Watch

### ISSUE THREAD

**Issue by blackreign** [WaveHack/OpenDominion#227]
### Current Behavior
Human (5% food bonus)
Gaia's Watch active
No food castle improvement
80 farms, no docks

Production advisor:
Food: | +7,544
Food Eaten: | -7,553
Food Decayed: | -275

**Net Change /hr
Food: | -284**

**Before hourchange:**
Food: | 27,487
**After hourchange:**
Food: | 26,547

### Expected Behavior
After hourchange:
Food: | 27,203

### Steps to Reproduce
Check status, gaia's watch and production advisor. Then watch hourchange.

### Environment
production (beta)

### Context
Version: r1645 @ local (develop #c4ec9f0)

**Comment by blackreign** [comment]
Calculated it and it acts like Gaia's Watch is being ignored.
80 farms = 6400 raw
human gives 105%
prestige gives 102,5% on top
that's 6888
minus 7553 consumption
minus 275
gives -940, which is what happened

Before hourchange:
Food: | 27,487
After hourchange:
Food: | 26,547

**Comment by WaveHack** [comment]
Very strange

https://github.com/WaveHack/OpenDominion/blob/develop/tests/Feature/TickTest.php#L212-L262

**Comment by WaveHack** [comment]
I am an idiot.

I think I know what's going on. Lemme test

**Comment by WaveHack** [comment]
Yeah found the bug.

tl;dr When something asks the SpellCalculator for the active spells of a dominion, all subsequent calls (even for other dominions) use the cached result 🤦‍♀️ 

![image](https://user-images.githubusercontent.com/1611537/37233330-0fd5057e-23f3-11e8-89ad-f0211de4a778.png)

**Comment by WaveHack** [comment]
Fixed!
